_posts/2020-07-27-aluminum-0.4.0.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Aluminum 0.4.0 Released"
+categories: release
+---
+
+[Aluminum](https://github.com/LLNL/Aluminum) provides a generic interface to high-performance communication libraries with a focus on allreduce algorithms. Blocking and non-blocking algorithms and GPU-aware algorithms are supported. Aluminum also contains custom implementations of select algorithms to optimize for certain situations. The latest release includes support for AMD GPUs using HIP/ROCm/RCCL.
```

Comment:
Is `allreduce` the right spelling? Should it be `all-reduce` or something else? (I honestly don't actually know)

Reply:
allreduce, or possibly Allreduce, is correct. See e.g. https://www.mpich.org/static/docs/latest/www3/MPI_Allreduce.html

Reply:
👍